scripts/selfheal.sh

```diff
@@ -16,38 +16,43 @@ string_to_seconds() {
   esac
 }
 
-INTERVAL=$(string_to_seconds ${SELFHEAL_INTERVAL})
-TIMEOUT=$(string_to_seconds ${SELFHEAL_TIMEOUT})
+INTERVAL=$(string_to_seconds 1m)
+TIMEOUT=$(string_to_seconds 1m)
 START_PERIOD=$(string_to_seconds ${SELFHEAL_START_PERIOD})
-
-RETRIES_REMAINING=${SELFHEAL_RETRIES}
+RETRIES_REMAINING=2
```

Comment:
what is the benefit of hardcoding this instead of having it configurable with defaults?